pkg/codegen/nodejs/gen.go

```diff
@@ -1905,12 +2008,13 @@ func (mod *modContext) gen(fs fs) error {
 	// Nested types
 	// Importing enums always imports inputs and outputs, so if we have enums we generate inputs and outputs
 	if len(mod.types) > 0 || (mod.pkg.Language["nodejs"].(NodePackageInfo).ContainsEnums && mod.mod == "types") {
-		input, output, err := mod.genTypes()
+		files, err := mod.genTypes()
 		if err != nil {
 			return err
 		}
-		fs.add(path.Join(modDir, "input.ts"), []byte(input))
```

Comment:
At first I was thinking it's breaking but the new code seems to gen input.ts too. Interesting.